src/main.py

```diff
@@ -1,14 +1,13 @@
 from waitress import serve
-
+from flask_apscheduler import APScheduler
 from app import config, create_app, db, logger
 from app.feeds import add_or_refresh_feed
 from app.models import Feed
+import global_ctx # global context, death to flask app context errors
 
-app = create_app()
-
-
-def port_over_old_feeds() -> None:
-    with app.app_context():
+def port_over_old_feeds():
```

Comment:
lint is going to be mad about this I think. I will run CI for you but if you want a preview you should be able to use the `scripts/ci.sh`.